crates/bitcoin-da/src/helpers/compression.rs

```diff
@@ -0,0 +1,16 @@
+use std::io::Write;
+
+#[cfg(feature = "native")]
```

Comment:
Actually you may put a whole module behind cfg feature=native, so we don't even have compression module on zkvm.

Reply:
da verifier uses `decompress_blob`